src/components/TheSideNavMenu.vue

```diff
@@ -4,8 +4,8 @@
     <!-- LIST ITEMS CONTAINER-->
     <v-list dense nav>
       <!-- HOME -->
-      <router-link to="/home" v-slot="{ href, route, navigate, isActive, isExactActive }">
-        <v-list-item @click="navigate">
+      <router-link to= '/home' v-slot="{href, route, navigate, isActive, isExactActive }">
+        <v-list-item  @click="scrollToHome()">
```

Comment:
there's no need to use another function, try to reuse the one that handles the routing for the menu items

Reply:
please, check the file again. I solved the issue